scripts/pi-hole/php/loginpage.php

```diff
@@ -30,34 +30,36 @@
           <input type="password" id="loginpw" name="pw" class="form-control" placeholder="Password" autofocus>
           <span class="fa fa-key form-control-feedback"></span>
         </div>
+        <div class="row">
+          <div class="col-xs-12 col-md-12">
+            <button type="submit" class="btn btn-primary form-control"><i class="fas fa-sign-in-alt"></i>&nbsp;&nbsp;&nbsp;Log in</button>
+          </div>
+        </div>
+        <br>
```

Comment:
Why is there a `<br>` between the rows? To get some extra spacing? Maybe we should use some `margin-bottom` on the upper element (the `row` div). I think this is more how it should be done.

Reply:
I just use the same as the existing `<br>` here:

https://github.com/pi-hole/AdminLTE/pull/1583/files#diff-22686796f3faafaceb91a6ff8a199efbL50